modules/lang-painless/src/main/java/org/elasticsearch/painless/Def.java

```diff
@@ -322,30 +324,30 @@ static MethodHandle lookupMethod(Definition definition, Lookup lookup, MethodTyp
       * This is just like LambdaMetaFactory, only with a dynamic type. The interface type is known,
       * so we simply need to lookup the matching implementation method based on receiver type.
       */
-    static MethodHandle lookupReference(Definition definition, Lookup lookup, String interfaceClass,
-            Class<?> receiverClass, String name) throws Throwable {
-         Class<?> interfaceType = definition.getJavaClassFromPainlessType(interfaceClass);
-         Method interfaceMethod = definition.getPainlessStructFromJavaClass(interfaceType).functionalMethod;
+    static MethodHandle lookupReference(PainlessLookup painlessLookup, Lookup lookup, String interfaceClass,
```

Comment:
I know this was already existing, but can you rename the MethodHandle.Lookup parameter to `methodLookup`? (maybe also don't static import Lookup)

Reply:
Done.